spec/controllers/oral_history_requests_controller_spec.rb

```diff
@@ -179,7 +179,7 @@
             }
 
             expect(response).to redirect_to(oral_history_requests_path)
-            expect(flash[:success]).to match /The files you requested are immediately available, from: #{Regexp.escape work.title}/
+            expect(flash[:notice]).to match /The files you requested from '#{Regexp.escape work.title}' are immediately available./
```

Comment:
OK, the reason I had it the other way first, is the name of the Oral History can be quite long in some cases.  I wanted them to see "are immediately available", which is the actually important content, before getting lost in the really long OH title. 

Your motivation to change it is just to make it a sentance without colon?   I guess if you've considered my point and still prefer it that way, I'll leave it to you, just mark this resolved and go ahead and merge?

Reply:
Yeah, I'm going to argue that the single quotes make it clear that the title is a title, no matter how long. I sincerely believe this is better copy, and I've cleared it with OH staff.

Reply:
OK, thanks!

It's not that I thought it wasn't clear it was a title, it's that I was worried the reader skimming might never make it to the most important words of the sentence, the whole point of the alert, telling the user that their files are immediately available!